.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore common IDE files
+/.idea
```

Comment:
Thanks Tom I've no idea why I don't do that. I already put .DS_Store and others in there; the thought just never occured to me!